### DISCUSSION TITLE
My laptop is frozen after using Human Demo for a couple of minutes

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#183] [Q&A] [answered]
After using Human Demo for a couple of minutes my laptop  -Macbook- is frozen, restarted automatically, and reported that chrome is running out of memory with only the demo tab!

**Comment by YaraAmin** [comment]
I checked this [issue](https://github.com/vladmandic/human/issues/167) but I can't find chrome 96 or newer version than 96

**Comment by vladmandic** [comment]
im sorry about that, its a really annoying issue  
chrome 96 is in google's *canary* channel, should be released in few weeks  
you can get it now from <https://www.google.com/chrome/canary/>

**Comment by YaraAmin** [comment]
Thanks!

**Comment by vladmandic** [comment, marked as answer]
issue #167 has been updated with full root-cause-analysis and possible workarounds  
new version of human 2.3.1 with a workaround for affected chrome versions has been released (ahead of time so not enough testing, but this is a really wide-spread issue)
